packages/config/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@kumahq/config",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Shared dependencies and configuration for kuma-gui",
+  "author": "Kong",
+  "type": "module",
+  "main": "./src/index.cjs",
+  "module": "./src/index.ts",
+  "types": "./src/index.ts",
+  "exports": {
+    ".": {
+      "require": "./src/index.cjs",
+      "import": "./src/index.ts"
+    }
+  },
+  "dependencies": {
+    "@bahmutov/cypress-esbuild-preprocessor": "^2.2.3",
+    "esbuild": "^0.24.0"
```

Comment:
I think `esbuild` should go into `devDependencies`? 🤔

Reply:
This is a dependency of `@kumahq/config`, its required to use the module if you are depending on it i.e. its a runtime requirement.

If we only need a dependency for the development of `@kumahq/config` specifically, then yes it should go in `devDepenendencies` here, but that is not the case for `esbuild`, its a dependency required by `@kumahq/kuma`.

For example, if we were to move `esbuild` to `devDependencies` and I `npm install`ed `@kumahq/kuma` from another project, then `esbuild` wouldn't be installed, because we've said "this is a dependency that is only required if you are developing `@kumahq/kuma`"